### DISCUSSION TITLE
joomla5: There is no import and export data button...

### DISCUSSION THREAD

**Discussion by behsoftinfo** [joomengine/Joomla-Component-Builder#964] [Q&A] [answered]
Hi
In old versions of JCB, in the components section, it was possible to export the component and import it.
There is no such option in the latest version 5. I have to export the old JCB project and import it into the new version. How should it be done?

I want to import the component that was in version 3 of JCB3, which was Joomla 3, into Joomla 5 and JCB5.

**Comment by Llewellynvdm** [comment, marked as answer]
Do manual upgrade from Joomla 3 to Joomla 4, then install the JCB for Joomla 4 and then upgrade to Joomla 5 and then install the JCB package for Joomla 5. This is the upgrade path now for JCB to Joomla 5. Do note that all versions of JCB can compile towards any version of Joomla, which means we can from Joomla 3 (JCB) build extensions for Joomla 5.

**Reply by Llewellynvdm** [reply]
The import export option is removed for now in JCB for Joomla 4 and 5... until further notice.

**Reply by jsr4id** [reply]
So how can i import the jcb packages and community packages?

**Reply by Llewellynvdm** [reply]
At the moment, JCB versions 4 and 5 do not include the package import facility, which was present in version 3. This absence is primarily due to time constraints and the encryption challenges that arose during Joomla's transition. We understand the importance of this feature, and we are actively working on reintroducing it or providing a similar solution.

For now, the recommended process to migrate your projects to Joomla 4 or 5 using JCB involves the following steps:

1. **Start with Joomla 3:** Begin your project in Joomla 3, where the package import facility is still available.
2. **Upgrade to Joomla 4:** Once your project is ready, upgrade your website to Joomla 4.
3. **Install Joomla 4 JCB:** After upgrading, install the Joomla 4 version of JCB.
4. **Upgrade to Joomla 5:** Next, update your website to Joomla 5.
5. **Install Joomla 5 JCB:** Finally, install the Joomla 5 version of JCB.

By following these steps, your project will be successfully migrated to Joomla 5. We have discussed this process multiple times in the forums, and we sincerely apologize for any inconvenience this may cause. Unfortunately, the situation is beyond our control, but we are committed to bringing back this feature because we rely on it heavily as well. The absence of this functionality in version 5 is a significant inconvenience for us too.

Thank you for your understanding and patience.
